src/geocel/VolumeSurfaceView.hh

```diff
@@ -72,7 +72,7 @@ VolumeSurfaceView::VolumeSurfaceView(SurfaceParamsRef const& params,
                                      VolumeId id)
     : params_(params), volume_(id)
 {
-    CELER_EXPECT(id < params.volume_surfaces.size());
+    CELER_EXPECT(id < params.volume_surfaces.size() || !params_);
```

Comment:
I don't think we should be allowed to create a volume surface view without having any surfaces 🤔

Reply:
My reasoning was we could still simulate optical photons without any optical surfaces (e.g. our simple LAr sphere), but we'd need to check for a surface when crossing a boundary. Or did we land on always requiring a volume with optical properties to have a surface?

Reply:
Sounds good, I'll warn and add invalid surface IDs if none are defined.